hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterClientProtocol.java

```diff
@@ -588,6 +589,19 @@ public long getPreferredBlockSize(String src) throws IOException {
     return rpcClient.invokeSequential(locations, method, Long.class, null);
   }
 
+  /**
+   * Use the combination of src locations and dst to return RemoteParam.
+   */
+  static private RemoteParam getRenameDestWithTrash(
```

Comment:
private static the other way.

Reply:
And again, why is this method called WithTrash? I don't see any trash related code in this method.